package.json

```diff
@@ -293,44 +307,34 @@
                     "group": "2_appServiceControlCommands@4"
                 },
                 {
-                    "command": "appService.DeployZipPackage",
+                    "command": "appService.OpenVSTSCD",
                     "when": "view == azureAppService && viewItem == appService",
                     "group": "3_appServiceDeployCommands@1"
                 },
-                {
-                    "command": "appService.LocalGitDeploy",
```

Comment:
You can remove `LocalGitDeploy` and `DeployZipPackage` completely correct? They're still in this file a few times